persistence-modules/hibernate5/pom.xml

```diff
@@ -88,6 +88,22 @@
             <artifactId>jaxb-api</artifactId>
             <version>2.3.0</version>
         </dependency>
+        <!-- validation -->
+        <dependency>
+	        <groupId>org.hibernate</groupId>
+	        <artifactId>hibernate-validator</artifactId>
+	        <version>${hibernate-validator.version}</version>
+        </dependency>
+	    <dependency>
```

Comment:
Let's please fix the whitespace here. Looks like you might have some tabs in here.

Reply:
Removed the tabs